INSTALL.md

```diff
@@ -68,8 +70,8 @@ System requirements:
 
 Synapse is written in Python but some of the libraries it uses are written in
 C. So before we can install Synapse itself we need a working C compiler and the
-header files for Python C extensions. See [Platform-Specific
-Instructions](#platform-specific-instructions) for information on installing
+header files for Python C extensions. See [Platform-specific
+prerequisites](#platform-specific-prerequisites) for information on installing
```

Comment:
Sorry, now that I look at the [rendered version](https://github.com/matrix-org/synapse/blob/4fcfacac4a1e3a31bfd69a2abbcd1ecc8e97567b/INSTALL.md#installing-from-source), we now have two links to the prerequisites next to each other. Even so, I still missed the original noticec when installing from source recently.

I think the line you've written is in a more visible spot. I would keep that, and move the following explanation:

> Synapse is written in Python but some of the libraries it uses are written in C. So before we can install Synapse itself we need a working C compiler and the header files for Python C extensions. ~~See Platform-specific prerequisites for information on installing these on various platforms.~~ (we can take this bit out)

into the top of the "Platform-specific prerequisites" section instead.

Reply:
Fixed in the latest commit.